package.json

```diff
@@ -10,6 +10,11 @@
     "base64",
     "gzip"
   ],
+  "dependencies": {
+    "stream":"~0.0.2",
+    "util": "~0.10.3",
+    "string_decoder": "~0.10.31"
```

Comment:
Erm, that looks like bundled front-end code of some sort? Perhaps compiled with webpack or browserify? Doesn't look like normal Node.js code that's for sure.

You probably just need to configure browserify/webpack correctly to include an appropriate core polyfill.

It would be much easier if you could explain the steps to reproduce